bql/builtin.go

```diff
@@ -133,10 +170,30 @@ func createFileSource(ctx *core.Context, ioParams *IOParams, params data.Map) (c
 		}
 	}
 
+	var repeat int64
+	if v, ok := params["repeat"]; ok {
+		r, err := data.AsInt(v)
+		if err != nil {
+			return nil, fmt.Errorf("'repeat' parameter must be an integer: %v", err)
+		}
+		repeat = r
+	}
+
+	var interval time.Duration
+	if v, ok := params["interval"]; ok {
```

Comment:
I see, that negative number interval does not cause an error and that specification policy is fine to me.